tests/test_read_only_trx.cpp

```diff
@@ -227,7 +229,9 @@ BOOST_AUTO_TEST_CASE(with_8_read_only_threads) {
 BOOST_AUTO_TEST_CASE(with_8_read_only_threads_no_tierup) {
    std::vector<const char*> specific_args = { "-p", "eosio", "-e",
                                              "--read-only-threads=8",
+#ifdef EOSIO_EOS_VM_OC_RUNTIME_ENABLED
```

Comment:
Why not have the `ifdef EOSIO_EOS_VM_OC_RUNTIME_ENABLED` enclose the whole test case?

Reply:
We still want this test to run on ARM without `"--eos-vm-oc-enable=none"`

Reply:
Wouldn't we run the same test twice then (without OC).

Reply:
yes